### PR TITLE
Comms nuget package

### PR DESCRIPTION
**Pull request by pallenSL**
FSharp.Core updated to fix Bristech.Srm.HttpConfig package

In Visual Studio this time, so
Lost of the changes are ^M.  Sorry.
packages.config has moved from a "<None Include" to a "<Content Include".
AWSSDK.Core has updated itself to 3.1.5.3.

### CODE REVIEW

File: Comms/Comms.fsproj [hunk 1]
```diff
@@ -35,27 +35,51 @@
     <Compile Include="DocsController.fs" />
     <Compile Include="Logging.fs" />
     <Compile Include="Program.fs" />
-    <None Include="packages.config" />
     <Content Include="app.config">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
     <EmbeddedResource Include="api.json" />
+    <Content Include="packages.config" />
```
Comment (lclarksl) [inline Comms/Comms.fsproj]: Why does this need to be changed to Content from None?

Comment (pallenSL) [inline Comms/Comms.fsproj]: It's not one I changed on purpose.  Reading up, it probably should be None.
One moment...  :D

**Comment by lclarksl** [comment]
:shipit:

**Comment by jebbinSCL** [comment]
:shipit:

**Pull request merged**